docs/ValidationCriteriaMaster.md

```diff
@@ -0,0 +1,764 @@
+
+</details>
+
+<details><summary>SEWI_06 details: Relative positioning verified</summary>
+
+
+> - Given an `IfcAnnotation/SUPERELEVATIONEVENT` or an `IfcAnnotation/WIDTHEVENT` having a linear placement relative to an alignment curve  
+> - This instance shall be verified according to the criteria in the [Product Relative Positioning](#Product-relative-positioning) section and the specification in the [Product Relative Positioning table](#Product-Relative-Positioning-table)
+
+</details>
+
+
+
+
+### Referent & Mileage
+
+| **ID**  | **CRITERIA**                                                                        | **VALUE [examples]**   | **ENTITY (if applicable)** | **CT (if applicable)**             |
+|---------|-------------------------------------------------------------------------------------|------------------------|----------------------------|------------------------------------|
+| REFE_00 | There are two nesting relationships for referents and for alignment components      |                        |                            |                                    |
+| REFE_01 | Relating Object and Distance Along for referent nesting are verified                | As per Referents Table | IfcReferent                | Object Nesting (IfcReferent usage) |
+| REFE_02 | Consistency between Stationing property's value and DistanceAlong attribute's value |                        | IfcReferent                | na                                 |
+
+> **Acceptance criteria**: For the **Referent nesting** capability, the validation procedure must verify that **all** the above validation criteria are satisfied.
+
+<details><summary>REFE_00 details: There are two nesting relationships for referents and for alignment components</summary>
+
+> - Given an `IfcAlignment`
+> - When `IfcAlignment` has an `IfcRelNests` relationship
+> - Then the `IfcRelNests` relationship is related either to an `IfcReferent` OR to an `IfcLinearElement`
+
+</details>
+
+<details><summary>REFE_01 details: Relating Object and Distance Along for referent nesting are verified</summary>
+
+> - Given a set of referents taken from the [Referents Table](#Referents-Table)
+> - Then the Referent, with the Referent Type and Name, exists
+> - And the Referent is placed relative to the Placement Product, using the requested Object Placement
+> - And the Referent is placed at the requested DistanceAlong
+
+</details>
+
+<details><summary>REFE_02 details: Consistency between Stationing property's value and DistanceAlong attribute's value</summary>
+
+> - Given an `IfcReferent`
+> - When `IfcReferent` has a property set named `Pset_Stationing`
+> - Then `Pset_Stationing` has a property named `Station`
+> - And the value of the `Station` property is equal to the value of the `DistanceAlong` attribute used to place the `IfcReferent`
```

Comment:
I wonder if this really can be a requirement? Or do I misunderstand? It is not a requirement that the Station is equal to the DistanceAlong (for example Station=32000 at DistanceAlong=0) and it is not a requirement that a Station "meter" equals a DistanceAlong "meter".

Reply:
You're right. I'll remove it